dev/source/docs/using-sitl-for-ardupilot-testing.rst

```diff
@@ -214,6 +214,43 @@ is equivalent to COM16:
 
 .. _using-sitl-for-ardupilot-testing_sitl_without_mavproxy_tcp:
 
+Swarming with SITL
+------------------
+
+SITL has support for launching multiple vehicles from a single command.
+With this method, SITL is restricted to launching vehicles of the same type.
+
+When launching multiple vehicles, each one will need a unique SysId parameter: :ref:`SYSID_THISMAV<SYSID_THISMAV>`.
+The easiest method to avoid SysId conflicts is with the ``--auto-sysid`` option.
+
+The number of vehicles is set with ``--count`` option.
+
+Up to this point, all the vehicles would spawn in the same default location and intersect.
+
+To avoid that, they must be spawned in unique locations.
+There are two primary approaches, using either a swarm offset line or a swarm configuration file.
+
+When using a swarm offset line, the option ``--auto-offset-line 90,10`` will space the 
+vehicles out at a line with heading of 90 degrees orientation. The vehicles will be spaced 10
+meters apart. Thus, they will be spread out east-west.
+
+Putting it all together for five Copter vehicles on an auto-offset-line:
+
+::
+
+    sim_vehicle.py -v Copter --map --console --count 5 --auto-sysid --auto-offset-line 90,10
+
+
+The other way to spawn the vehicles is the swarm configuration file.
+An example configuration is found in ``Tools/autotest/swarminit.txt``, with the file format described in the header.
+This allows configurable ENU offsets as well as an initial absolute heading for each vehicle.
+
+::
+
+    sim_vehicle.py -v Copter --map --console --count 5 --auto-sysid --swarm Tools/autotest/swarminit.txt
+
+:ref:`MavProxy's Multiple Vehicle Guide <multi>` contains information to control the swarm.
```

Comment:
This doesn't link. Any syntax tips?

Reply:
I would do the MAVProxy separate...merge it...to establish the links inventory, then the ref in this doc can be tested locally

Reply:
Done.

Reply:
I've already done the change and merged it....I think the wiki on the server needs to rebuild for the inventory to update, but will trying building this now anyway

Reply:
yep will have to wait for wiki to build to really test the link to the MAVProxy mult page....

Reply:
Ok. Want to merge this as-is, then add the cross-ref later?

Reply:
it can wait a day